IdentityCore/src/webview/response/MSIDWebOAuth2AuthCodeResponse.m

```diff
@@ -38,10 +38,11 @@ - (instancetype)initWithURL:(NSURL *)url
     
     if (self)
     {
+        if (self.oauthError) return self;
```

Comment:
Do we intend to return self here. If so can we add a log comment?

Reply:
yes, we do want to return response object with error code inside. Regarding logging, we do log when error is created in the parent class.